snapcraft/storeapi/v2/_api_schema.py

```diff
@@ -21,6 +21,128 @@
 
 from typing import Any, Dict
 
+
+RELEASES_JSONSCHEMA: Dict[str, Any] = {
```

Comment:
Side note, It would appear Final typing does not affect a dictionary.

Reply:
side side note, we should enable pyright, does a better static job and is faster than mypy